scripts/login.sh

```diff
@@ -0,0 +1,114 @@
+#!/bin/bash
```

Comment:
Yes, I did this before the Go implementation. Then I realized that it was not going to work everywhere and I did the Go script, that is probably more maintanable, and readable. I'll move this in the `docs` just because it took me a lot of time to do it. :laughing: